repo-infra/verify/boilerplate/BUILD

```diff
@@ -0,0 +1,3 @@
+package(default_visibility = ["//visibility:public"])
+
+exports_files(glob(["*.txt"]))
```

Comment:
@martinmaly can you elaborate on your "makefile" comment? I'm not sure why the location of the tools changes anything - once we git clone (and run glide) - its all just files in our src tree.